### PR TITLE
Fix: Example3 calls glViewport too early.

### PR DESCRIPTION
**Pull request by lmurmann**
In example3, glViewport was called before initializing GLAD, causing the example to crash on Windows. Just removing the glViewport call seems to work fine.

### CODE REVIEW

File: src/example3.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ int main(int /* argc */, char ** /* argv */) {
 
     int width, height;
     glfwGetFramebufferSize(window, &width, &height);
-    glViewport(0, 0, width, height);
```
Comment (wjakob) [inline src/example3.cpp]: Probably this should be moved later rather than outright deleting it?

Comment (lmurmann) [inline src/example3.cpp]: It is called on every frame in Screen::drawWidgets. The glViewport call in the initialization code seems unneccessary. Actually, the two lines above
```
int width, height;
glfwGetFramebufferSize(window, &width, &height);
```
can be deleted as well, without visible side-effects. These two can of course also be left in to demonstrate the API usage.

Comment (wjakob) [inline src/example3.cpp]: This is incorrect: the point of these lines is to clear the screen once before the mainloop. Application startup often takes a second or so (e.g. to load textures and other resources), and annoying garbage can be seen in the meantime on some platforms without this step.

**Comment by lmurmann** [comment]
Ah, interesting about clearing before entering the main loop. Although I don't see how the three lines above contribute to a framebuffer clear.

Wouldn't you just clear the framebuffer with 
```
glClearColor() -> glClear() -> glfwSwapBuffers() ?
```

I added this sequence right after `Screen::initialize`.

**Comment by wjakob** [comment]
should be fixed by #141. Ignore what I said about clearing the screen.

**Pull request closed without merging**